spec/android_unit_test_checker_spec.rb

```diff
@@ -334,7 +348,7 @@ def prefix_text_lines(content:, prefix:)
     end
 
     def expect_class_names_match_report(class_names:, error_report:)
-      expect(class_names.length).to eq(error_report.length)
+      # expect(class_names.length).to eq(error_report.length)
```

Comment:
Hmm why is this check failing? 🤔

Reply:
Oh sorry, it was from me testing things. I'll revert this change.

Reply:
Fixed in https://github.com/Automattic/dangermattic/pull/92/commits/50ba674e7d09e771de12ff5eb0467247ffbbb14c